src/css/style.css

```
@@ -7,6 +7,32 @@
   --button-light-blue: #76A4FF;
   --input-border-blue: #3878F9;
   --gray-op-2: rgb(139, 148, 158, 0.2);
+  --radius: 10px;
+}
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+.d-none {
+  display: none !important;
+}
+.mt-2 {
+  margin-top: 0.75rem !important;
+}
+.mt-3 {
```

Let's to use Tailwindcss classes how reference https://tailwindcss.com/docs/margin

```suggestion
.mt-4 {
```

Classes | Value
-- | --
m-4 | margin: 1rem; /* 16px */